_posts/2019-10-10-bazel-1.0.md

```diff
@@ -0,0 +1,95 @@
+*   [`--incompatible_no_target_output_group`](https://github.com/bazelbuild/bazel/issues/7949)
+*   [`--incompatible_remove_legacy_whole_archive`](https://github.com/bazelbuild/bazel/issues/7362)
+*   [`--incompatible_require_java_toolchain_header_compiler_direct`](https://github.com/bazelbuild/bazel/issues/7990)
+*   [`--incompatible_restrict_attribute_names`](https://github.com/bazelbuild/bazel/issues/6437)
+*   [`--incompatible_restrict_named_params`](https://github.com/bazelbuild/bazel/issues/8147)
+*   [`--incompatible_skip_genfiles_symlink`](https://github.com/bazelbuild/bazel/issues/8651)
+*   [`--incompatible_tls_enabled_removed`](https://github.com/bazelbuild/bazel/issues/8061)
+*   [`--incompatible_use_aapt2_by_default`](https://github.com/bazelbuild/bazel/issues/6907)
+*   [`--incompatible_use_native_patch`](https://github.com/bazelbuild/bazel/issues/8316)
+*   [`--incompatible_use_specific_tool_files`](https://github.com/bazelbuild/bazel/issues/8531)
+*   [`--incompatible_windows_bashless_run_command`](https://github.com/bazelbuild/bazel/issues/8240)
+*   [`--incompatible_windows_native_test_wrapper`](https://github.com/bazelbuild/bazel/issues/6622)`
+
+
+## Windows
+
+
+
+*   Genrule now supports `cmd_bash`, `cmd_ps`, `cmd_ba`t attributes. More details at [https://docs.bazel.build/versions/master/be/general.html#genrule.cmd](https://docs.bazel.build/versions/master/be/general.html#genrule.cmd)
+*   C++: you can now get a generated DEF file from the `def_file` output group of cc_library.
+*   MSYS2 / Bash: testing targets ("bazel test //foo"), running binary targets ("bazel run //bar"), and fetching repository rules no longer require MSYS2. See (respectively) the [`--[no]incompatible_windows_native_test_wrapper`](https://github.com/bazelbuild/bazel/issues/6622), [`--[no]incompatible_windows_bashless_run_command`](https://github.com/bazelbuild/bazel/issues/8240), and [`--[no]incompatible_use_native_patch`](https://github.com/bazelbuild/bazel/issues/8316) flags: all are enabled by default.
+
+
+## Execution
+
+
+
+*   Set `--experimental_allow_tags_propagation` flag to propagate tags to the action's execution requirements from targets. Such tags should start with: no-, requires-, supports-, block-, disable-, cpu:. See [#8830](https://github.com/bazelbuild/bazel/issues/8830) for details.
+*   All rules now have a default `exec_properties` attribute just like the one on a [platform](https://docs.bazel.build/versions/master/be/platform.html#platform.exec_properties) rule. 
+*   All gRPC connections from Bazel will default to TLS enabled.  To disable TLS use the grpc:// scheme in your URIs. Flags affected are: `--remote_cache`, `--remote_executor` and `--bes_backend`. See #8061 for details.
+*   Make either `--subcommands` or `--verbose_failures` imply `--materialize_param_files`
+
+
+## Configurability
+
+
+
+*   `config_setting` can now check multiple values on "--foo=firstVal --foo=secondVal ..."-style flags
```

Comment:
config_setting should link to https://docs.bazel.build/versions/master/be/general.html#config_setting

Reply:
Done